src/main/fc/runtime_config.c

```diff
@@ -53,7 +53,8 @@ const char *armingDisableFlagNames[]= {
     "MSP",
     "PARALYZE",
     "GPS",
-    "ARMSWITCH"
+    "ARMSWITCH",
```

Comment:
Needs to be the other way round, see `runtime_config.h`.

Reply:
Oops!